src/Yasumi/Provider/UnitedKingdom.php

```diff
@@ -203,6 +204,33 @@ protected function calculatePlatinumJubileeBankHoliday(): void
         ));
     }
 
+    /**
+     * Queen Elizabeth II’s funeral is an extra bank holiday added on 10 September 2022
```

Comment:
10 September 2022 > 19 September 2022

Reply:
@sylvaincombes The 10th is the date the bank holiday was announced

Reply:
oh ok sorry I misunderstood and thought it was a typo

Reply:
Thank you for your work, hope it will be merged soon